.github/actions/cargo-risczero-install/action.yaml

```diff
@@ -73,10 +78,15 @@ runs:
     - name: install cargo-risczero from crates.io
```

Comment:
Do we actually still need to install cargo-risczero at all? Hasn't it been completely replaced by rzup?